app/visualisation_methods/pareto/plugin.rb

```diff
@@ -70,30 +51,59 @@ def get_pareto_data(moes)
         maxes[arg_name] = params[arg_name].max
       end
 
+      #preparing Chart content
+      data =[]
```

Comment:
Aha, no to w takim razie przy mergowaniu tego do development trzeba rozwiązać konflikty związane z tym refactoringiem na korzyść zrefaktorowanej wersji.